connector_prestashop/unit/importer.py

```diff
@@ -492,7 +493,12 @@ def _after_import(self, binding):
 def import_batch(session, model_name, backend_id, filters=None, **kwargs):
     """ Prepare a batch import of records from PrestaShop """
     backend = session.env['prestashop.backend'].browse(backend_id)
-    env = backend.get_environment(model_name, session=session)
+    connector_env_options = kwargs.get('connector_env_options', False)
+    if connector_env_options:
+        env = backend.get_environment(
+            model_name, session=session, **connector_env_options)
+    else:
+        env = backend.get_environment(model_name, session=session)
```

Comment:
Why not add attributes to the `ConnectorEnvironment` with the `connector_env_options`? This way you wouldn't need a custom ConnectorEnvironment